packages/playground/windows/playground-win32/packages.config

```diff
@@ -6,9 +6,9 @@
       Playground-Win32 is an unpackaged win32 app, and needs to use Microsoft.UI.Xaml from a prerelease, to be able to carry WinUI in-app instead of using the Framework Package 
     /WARNING
   -->
-  <package id="Microsoft.UI.Xaml" version="2.7.0-prerelease.210913003" targetFramework="native"/>
```

Comment:
This change needs to be reverted. playground-win32 is still using MUX, not WinAppSDK.